packages/jest-resolve-dependencies/src/__tests__/dependency_resolver.test.ts

```diff
@@ -33,10 +33,10 @@ beforeEach(() => {
     roots: ['./packages/jest-resolve-dependencies'],
   });
   return Runtime.createContext(config, {maxWorkers, watchman: false}).then(
-    (hasteMap: any) => {
+    (runtimeContext: any) => {
```

Comment:
This was never actually a `HasteMap` instance lol, `HasteMap`s don't even have a `.resolver` property 😄